int32 laps_count
float32 altitude
float32 angular_velocity
---
